app/src/interfaces/boolean/boolean.vue

```diff
@@ -45,6 +49,30 @@ export default defineComponent({
 			default: '#00C897',
 		},
 	},
+	methods: {
```

Comment:
Would you be ok to change this to Composition API? Just for consistency sake 👍 

The `input.vue` should be a good reference for this.

Reply:
Done!

Reply:
Thanks!